Source/Tools/LinuxEmulation/ArchHelpers/MContext.h

```diff
@@ -123,6 +123,10 @@ static inline uint64_t GetPc(void* ucontext) {
   return GetMContext(ucontext)->pc;
 }
 
+static inline uint64_t* GetArmPc(void* ucontext) {
+  return reinterpret_cast<uint64_t*>(&GetMContext(ucontext)->pc);
```

Comment:
Perhaps make getpc etc return a reference?

Reply:
If this was switched over to a reference then it would still need a reinterpret_cast anyway because mcontext type for member `pc` is `unsigned long` (I think it differs between types in different aarch64 libraries even). So it's less filthy to just cast to uint64_t*.